armi/physics/neutronics/crossSectionGroupManager.py

```diff
@@ -852,15 +852,19 @@ def createRepresentativeBlocksUsingExistingBlocks(
         ----------
         blockList : list
             A list of blocks defined within the core
-        originalRepresentativeBlocks : list
-            A list of unperturbed representative blocks that the new representative blocks are formed from
+        originalRepresentativeBlocks : dict
+            A dict of unperturbed representative blocks that the new representative blocks are formed from
+            keys: XS group ID (e.g., "AA")
+            values: representative block for the XS group
 
         Returns
         -------
         blockCollectionByXsGroup : dict
             Mapping between XS IDs and the new block collections
         modifiedReprBlocks : dict
             Mapping between XS IDs and the new representative blocks
+        origXSIDsFromNew : dict
+            Mapping of original XS IDs to new
```

Comment:
can we call out at a high level why it might have a new? something like 

`Mapping of original XS IDs to new XS ID typically due to some time of perturbation (eg: isotope temperature)`